lambdas/tests/unit/helpers/data/bulk_upload/expected_success_report.csv

```diff
@@ -1,11 +1,11 @@
-NhsNumber,UploaderOdsCode,Date
-9000000000,Y12345,2012-01-13
-9000000001,Y12345,2012-01-13
-9000000002,Y12345,2012-01-13
-9000000003,Y12345,2012-01-13
-9000000004,Y12345,2012-01-13
-9000000009,Z12345,2012-01-13
-9000000010,Z12345,2012-01-13
-9000000011,Z12345,2012-01-13
-9000000012,Z12345,2012-01-13
-9000000013,Z12345,2012-01-13
+NhsNumber,UploaderOdsCode,Date,RegisteredAtUploaderPractice
+9000000000,Y12345,2012-01-13,SUSP
+9000000001,Y12345,2012-01-13,DECE
+9000000002,Y12345,2012-01-13,REST
+9000000003,Y12345,2012-01-13,True
+9000000004,Y12345,2012-01-13,False
+9000000009,Z12345,2012-01-13,SUSP
+9000000010,Z12345,2012-01-13,DECE
+9000000011,Z12345,2012-01-13,REST
+9000000012,Z12345,2012-01-13,False
+9000000013,Z12345,2012-01-13,True
```

Comment:
Just going to have to trust that these are all correct, I'm assuming this has come from running the unit tests -> failed -> here's what it expected and you've dropped it in?